src/App.tsx

```diff
@@ -1,8 +1,26 @@
-import { verifiedFetch } from '@helia/verified-fetch'
+import { createVerifiedFetch } from '@helia/verified-fetch'
+import { createHelia } from 'helia'
```

Comment:
if you want no P2P or direct retrieval, you should be able to remove direct Helia and use this instead:

```suggestion
import { createHeliaHttp } from 'helia'
```